README.md

```diff
@@ -0,0 +1,40 @@
+Valkey RFC
+==========
+
+This is a collection of feature proposals and descriptions of changes to Valkey
+that require some more details than just the text in a pull request or an issue.
+It is loosely inspired by RFCs and by Python's enhancement proposals (PEP).
+
+Each feature or larger topic is described in a markdown file that's named in
+uppercase and ends in `.md`. These files are not formally numbered, but we use
+the pull request number that added an RFC to refer to the change. For example,
+this description in the README.md file was written in RFC #1.
+
+Workflow
+--------
+
+An RFC starts off as a pull request. It's reviewed for formatting, style and
+consistency. Then the proposal is merged. This doesn't mean that the feature is
```

Comment:
I think I would also include there is a standard for quality of an RFC on the content. If something is very vague and unclear, it's probably worth improving it before merging it. We won't reject it based off the proposal though.